build-run.ps1

```diff
@@ -33,18 +33,13 @@ cd $PSScriptRoot
 $repoFolder = $PSScriptRoot
 $env:REPO_FOLDER = $repoFolder
 
-if ($env:BuildRunner -eq "MyGet"){
-    Remove-Item -Force -Recurse (Join-Path $PSScriptRoot (Join-Path "test" "Pomelo.EntityFrameworkCore.MySql.PerfTests"))
-}
-
-$koreBuildZip="https://github.com/PomeloFoundation/KoreBuild/releases/download/1.0.0/KoreBuild-1.0.0.zip"
+$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/dev.zip"
```

Comment:
Are you sure the aspnet/KoreBuild/archive/dev.zip will clone the 1.1.0-preview1-final environment(not 1.1.0-*)?